help/sources/tutorials/ui/create/data-partners/acxiom-prospecting-data-import.md

```diff
@@ -0,0 +1,154 @@
+
+* **Search source fields** - Entering text into this field will restrict the existing map fields, showing only those that contain that substring.
+* **"All fields" filter fields** - Drop down  restricting the existing map fields, showing only those that match the status indicated in the drop down.  Options include:
+  * **All fields (Default)** - Display all mapping
+  * **Fields with errors** - Show only those that have validation errors
+  * **Required fields** - Show only those that are required
+  * **Identity fields** - Show only those that are defined as identity fields
+  * **Mapped fields** - Show only those that have been mapped
+  * **Unmapped fields** - Show only those that have not been mapped
+* **New field type**
+  * **Add new field** - New fields can be appended, allowing a user to duplicate a source column to a different target field, or adding fields that were previously discarded
+  * **Add calculated field** - Allows field manipulation though scripting.  [Field mapping functions in the UI](../../../../../data-prep/functions.md)
+* **Import Mapping** - This allows mapping to be imported from previous dataflow configurations.  When selected a list will be presenting with the dataflows, their last run status, date of processing, as well as the source used.
+* **Preview Data** - This option will create an example of the mapping as presented.  A filter option allows to restrict the result to specific columns or outlining columns presenting errors.
+* **Validate** - This repeats the validation of the current mapping presented, ensuring that fields qualify to the schema selected.
+* **"Clear all" mapping** - Resets the mapping, removing all entries.
+<br>![Mapping](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-mapping.png)
+
+## Scheduling ##
+Defines when the dataflow should be run.
+
+* **Frequency** - Indicates how often the flow should be run, once a week, day, hour, minute.  Or simply to only run once.
+* **Interval** - When a Frequency is selected other than once, interval to establish a set time frame between every ingestion. For example, an ingestion frequency set to Day and an interval set to 15 means that your dataflow is scheduled to ingest data every 15 days.
+* **Start Time** - The timestamp for the projected run, presented in UTC time zone.
+* **Backfill** - A boolean value that determines what data is initially ingested. If backfill is enabled, all current files in the specified path will be ingested during the first scheduled ingestion. If backfill is disabled, only the files that are loaded in between the first run of ingestion and the start time will be ingested. Files loaded prior to start time will not be ingested.
+<br>![Schedualing](../../../../images/tutorials/create/acxiom-prospect-suppression-data-sourcing/image-source-scheduling.png)
+
+## Review ##
```

Comment:
- Remove extra ##
- Revise to "Review your dataflow"

Reply:
Cleaned up extra ## and changed title to suggested wording.